vignettes/details.Rmd

```diff
@@ -22,3 +22,5 @@ knitr::opts_chunk$set(
 * You want to cache the compilation across sessions
 
 If you set this environment variable, then we'll create models here, each with a name in the pattern `dust_<hash>` where `<hash>` is the hash of the generated code.  This means that if your model changes we'll still recompile it.
+
+`DUST_QUIET`: an environment which controls the default level of verbosity when compiling dust models.  Set this to a truthy value (e.g., `TRUE`) to change the default.  This can always be overridden by providing a value for the `quiet` argument to `dust_compile` and `dust_package`.
```

Comment:
```suggestion
`DUST_QUIET`: an environment variable which controls the default level of verbosity when compiling dust models.  Set this to a truthy value (e.g., `TRUE`) to change the default.  This can always be overridden by providing a value for the `quiet` argument to `dust_compile` and `dust_package`.
```